projects/netgrif-components-core/src/lib/forms/models/abstract-registration.component.ts

```diff
@@ -40,7 +39,11 @@ export abstract class AbstractRegistrationComponent implements HasForm, OnDestro
 
     protected constructor(protected _signupService: SignUpService,
                           protected _log: LoggerService,
-                          protected _translate: TranslateService) {
+                          protected _translate: TranslateService,
+                          @Optional() @Inject(NAE_MIN_PASSWORD_LENGTH) protected minPasswordLength: number | null) {
+        if (!minPasswordLength) {
+            this.minPasswordLength = 8;
```

Comment:
fi the same default value is used in both registration and change password components, perhaps, it should be taken from one place

Reply:
Done